src/Repository.js

```diff
@@ -118,8 +120,22 @@ export default class Repository {
   }
 
   buildPackageGraph() {
-    this._packages = PackageUtilities.getPackages(this);
-    this._packageGraph = PackageUtilities.getPackageGraph(this._packages);
+    const packages = PackageUtilities.getPackages(this);
+    const packageGraph = PackageUtilities.getPackageGraph(packages, false, this.lernaJson.useGitVersion
+      ? new GitVersionParser(this.lernaJson.gitVersionPrefix)
+      : null);
+
+    if (this.lernaJson.useGitVersion) {
+      packages.forEach((pkg) => {
+        pkg.versionSerializer = new VersionSerializer({
+          monorepoDependencies: packageGraph.get(pkg.name).dependencies,
```

Comment:
`graphDependencies` sounds clearer to me, here. It's not the dependencies of the entire monorepo, after all.